plugin.video.bfi/addon.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.bfi" name="BFI Player" version="0.0.1" provider-name="fraser">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.routing" version="0.2.0"/>
+        <import addon="script.module.beautifulsoup4" version="4.3.2" />
+        <import addon="script.module.requests" version="2.12.4"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="main.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Search and play free movies from the BFI collection</summary>
+        <description lang="en">This add-on enables playing of videos and movies from the BFI Player website. It is not created, maintained or in any way affiliated with the British Film Institute. The add-on only provides an interface to access the free content on the BFI Player website from Kodi.</description>
+        <language></language>
+        <platform>all</platform>
+        <license>MIT</license>
+        <forum></forum>
+        <website>www.fraserchapman.co.uk</website>
+        <email>fraser.chapman@gmail.com</email>
+        <source></source>
```

Comment:
Please add a source (it is linked from the addon website and wiki) pointing to your github